src/targets/browser/index.jsx

```diff
@@ -94,27 +93,25 @@ const App = props => {
           dictRequire={lang => require(`photos/locales/${lang}`)}
         >
           <CozyProvider client={props.client}>
-            <DataProxyProvider>
```

Comment:
So this means no dataproxy at all even if not public?

Reply:
Sorry about that, I did some cleanup after testing and I mixed up files... It is now in public target  as it should.
That being said, there is currently no search for non-public pages , not sure why: https://github.com/cozy/cozy-photos/blob/master/src/photos/components/Layout.jsx#L115